sdks/go/test/integration/io/fhirio/fhirio_test.go

```diff
@@ -153,6 +153,7 @@ func populateStore(storePath string) []string {
 			resourcePaths = append(resourcePaths, resourcePath)
 		}
 	}
+	time.Sleep(time.Second) // give some time for data to propagate. prevents flaky results
```

Comment:
Now that we've added search... If indexing is overloaded/slow, this could take a lot longer than a second. Can we add a follow-up maybe to make the search test retry with a backoff? And maybe make the default here like 30 seconds, to prevent flakes.